swarm/fuse/swarmfs_unix.go

```diff
@@ -120,6 +120,10 @@ func (swarmfs *SwarmFS) Mount(mhash, mountpoint string) (*MountInfo, error) {
 
 	log.Trace("swarmfs mount: traversing manifest map")
 	for suffix, entry := range manifestEntryMap {
+		if suffix == "" { //empty suffix means that the file has no name - i.e. this is the default entry in a manifest. Since we cannot have files without a name, let us ignore this entry
+			log.Warn(fmt.Sprintf("Manifest has an empty-path (default) entry, which would case FUSE mount to fail. Ignoring entry %v", entry.Hash))
```

Comment:
would case -> would cause

Reply:
* use the structured logging. no need to use `Sprintf`
* use a simpler message
* better put the context, like what hash is being mounted or under what mountpoint. the entry's hash is the least relevant

`log.Warn("Ignore manifest entry with empty path",  "mount point", mountpoint, "swarm hash", hashToMount)`

Reply:
> use the structured logging.

I don't know what that means. I copied the Sprintf syntax from elsewhere in the code.

>  the entry's hash is the least relevant

how so? This hash represents the one piece of data that is missing from the fuse mount.


I'll make a simpler message with no info.